src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,111 @@
+                manufacturer.setId(id);
+            }
+        } catch (SQLException throwables) {
+            throw new DataProcessingException("Can't create new manufacturer into DB. Manufacturer "
+                    + manufacturer, throwables);
+        }
+        return manufacturer;
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        String selectQuery = "SELECT * FROM manufacturers "
+                + "WHERE id = ? AND is_deleted = FALSE";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement getManufacturerStatement = connection
+                         .prepareStatement(selectQuery)) {
+            getManufacturerStatement.setLong(1, id);
+            ResultSet resultSet = getManufacturerStatement.executeQuery();
+            if (resultSet.next()) {
+                return Optional.of(getManufacturer(resultSet));
+            }
+        } catch (SQLException throwables) {
+            throw new DataProcessingException("Can't get manufacturer by id " + id, throwables);
+        }
+        return Optional.empty();
+    }
+
+    @Override
+    public List<Manufacturer> getAll() {
+        List<Manufacturer> list = new ArrayList<>();
+        String slelectQuery = "SELECT * FROM manufacturers WHERE is_deleted = FALSE";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 Statement getAllManufacturerStatement = connection.createStatement()) {
+            ResultSet resultSet = getAllManufacturerStatement.executeQuery(slelectQuery);
+            while (resultSet.next()) {
+                list.add(getManufacturer(resultSet));
+            }
+        } catch (SQLException throwables) {
+            throw new DataProcessingException("Can't get all data from DB", throwables);
+        }
+        return list;
+    }
+
+    @Override
+    public Manufacturer update(Manufacturer manufacturer) {
+        String updateQuery = "UPDATE manufacturers SET name = ?,"
+                + " country = ? WHERE id = ? AND is_deleted = FALSE";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement updateManufacturerStatement = connection
+                        .prepareStatement(updateQuery)) {
+            updateManufacturerStatement.setString(1, manufacturer.getName());
+            updateManufacturerStatement.setString(2, manufacturer.getCountry());
+            updateManufacturerStatement.setLong(3, manufacturer.getId());
+            updateManufacturerStatement.executeUpdate();
+        } catch (SQLException throwables) {
+            throw new DataProcessingException("Can't update manufacturer by id "
+                    + manufacturer.getId(), throwables);
+        }
+        return manufacturer;
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String deleteQuery = "UPDATE manufacturers SET is_deleted = 1 WHERE id = ?";
```

Comment:
please see checklist
```suggestion
        String deleteQuery = "UPDATE manufacturers SET is_deleted = true WHERE id = ?";
```